### ISSUE TITLE
Strange problem with gvim and Ubuntu 20.04

### ISSUE THREAD

**Issue by ayonganguly** [vim-latex/vim-latex#177]
I am using Ubuntu 20.04 and vim-latex. I am also using plug plugin manager. When I am opening a .tex file using gvim, Super+Right (view split on right) and Super+Left (view split on left) are not working. Though Super+Up and Super+Down are working. Also Super+Right and Super+Left are working other file types (like .R, .txt). These keyboard shortcuts are working well when I am opening .tex file using vim in terminal. Please look into it and suggest me a remedy.

**Comment by gerw** [comment]
How do you bind the super+right key? Is something else happening when you press this key combination in a tex file?

**Comment by ayonganguly** [comment]
Thanks for replying. It is bound using ubuntu setting (screenshot is attached). Nothing are happening when I am pressing the key combo.I hope this answers your questions.
![Screenshot from 2020-05-01 15-40-37](https://user-images.githubusercontent.com/63195834/80798491-24784700-8bc2-11ea-900d-b371f0e3414a.png)

**Comment by gerw** [comment]
Do I get it correctly that these are system-wide shortcuts? Then this rather seems to be a bug of your window manager since applications should not be able to overwrite these shortcuts.

What happens if you type ":<ctrl+v><super+left>" in gvim?

**Comment by ayonganguly** [comment]
Yes, you are right. These are system wide shortcuts. Most of other system wide shortcuts are working excluding <Super+Right> and <Super+Left>. <Ctrl+v> is also working. Please note that if I disable vim-latex, all the shortcuts are working including <Super+Right> and <Super+Left>.

**Comment by gerw** [comment]
What happens if you press ":" + "<ctrl-v>" + "<super-left>" in gvim / vim resp.?

**Comment by ayonganguly** [comment]
Both in gvim and vim, the sequence is disappearing after pressing second + and nothing happen. I hope that this sequence need to press continuously, right?

**Comment by gerw** [comment]
My previous message was scrambled by github. No, you should not press "+", only the key ":", afterwards the combo `<ctrl-v>` and afterwards `<super-left>`.

**Comment by ayonganguly** [comment]
Nothing happen. ":" is disappearing when I am pressing <Ctrl-v>.

**Comment by gerw** [comment]
Sorry, I cannot reproduce your problem and cannot do anything about it. I suspect that some mapping of vim-latex suite collides with your mappings.

You could try to disable some features of vim-latex to check whether this helps (see, e.g., `texrc`).

**Comment by ayonganguly** [comment]
Well, I shall check. Thank you.
